elide-graphql/src/test/java/com/yahoo/elide/graphql/FetcherFetchTest.java

```diff
@@ -200,11 +200,6 @@ public void testAliasSameRelationship() throws Exception {
         runComparisonTest("aliasSameRelationship");
     }
 
-    @Test
```

Comment:
Why did we remove this test?

Reply:
This test checks if there is two root level query of same entity, (without alias) then the response should merge these two queries. We decided to not support these conditions. 
Request- 
`
{
  book(ids: ["1"]) {
    edges {
      node {
        alias1: id
      }
    }
  }
  book(ids: ["1"]) {
    edges {
      node {
        alias2: id
      }
    }
  }
}
`

Response-
`
{
  "book": {
    "edges": [
      {
        "node": {
          "alias1": "1",
          "alias2": "1"
        }
      }
    ]
  }
}
`